styles/index.css

```diff
@@ -1,3 +1,62 @@
+.overlay-container {
+  width: 100%;
+  height: 100vh;
+  position: relative;
+}
+.overlay-text {
+  position: absolute;
+  color: rgb(234, 234, 233);
+  top: 35%;
+  left: 5%;
+  font-family: var(--paragraph-sans);
+  font-weight: lighter;
```

Comment:
what does this do?

Reply:
I tried to get the text thinner so I used font-weight: lighter. But yes, the paragraph sans already set the weight to be 400 so there is no need for that.

Reply:
Yes that's correct. You can also check https://fonts.google.com/specimen/Open+Sans for full specifications on the Font. Currently we have only imported 400 so no other weight can be applied to Open Sans. Even if you try to bold the character. I'd recommend to use 300/400/500etc value to determine font-weight. `lighter` can mean 200 or 300 unpredictably